src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,26 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import React from 'react';
+import classNames from 'classnames';
+
+import { User } from '../../types/User';
+
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  completed: boolean,
+  title: string,
+  user: User | null,
+};
+
+export const TodoInfo: React.FC<Props> = ({ completed, title, user }) => (
+  <li className={classNames(
+    'TodoInfo',
+    { 'TodoInfo--completed': completed },
+  )}
+  >
+    <h2 className="TodoInfo__title">
+      {title}
+    </h2>
+
+    <UserInfo user={user} />
```

Comment:
```suggestion
  {user &&  (<UserInfo user={user} />)}
```
let's render it when the user exists